<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RofloLisp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/ico(1).ico</normaloff>../../../Downloads/ico(1).ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>911</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>240</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>300</width>
       <height>271</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>298</width>
        <height>269</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEditcons">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>370</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>tabWidget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>tabWidget_2</zorder>
   <zorder>tabWidget</zorder>
   <zorder>tableWidget</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>textEditcons</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Открыть последние</string>
     </property>
     <addaction name="action_5"/>
    </widget>
    <widget class="QMenu" name="menu_8">
     <property name="title">
      <string>Настройки</string>
     </property>
     <addaction name="action_13"/>
    </widget>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="menu_7"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
    <addaction name="menu_8"/>
    <addaction name="separator"/>
    <addaction name="action_16"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="action_17"/>
    <addaction name="separator"/>
    <addaction name="action_19"/>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
    <addaction name="separator"/>
    <addaction name="action_23"/>
    <addaction name="action_24"/>
    <addaction name="separator"/>
    <addaction name="action_26"/>
    <addaction name="action_27"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_29"/>
    <addaction name="action_30"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Проект</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Отладка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Новый файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Новое окно</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Открыть файл...</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>какие-то</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Сохранить все</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Автосохранение</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>тоже чё-то</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Вырезать </string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>Заменить</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>Найти в файлах</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>Заменить в файлах</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>Файлы проекта</string>
   </property>
  </action>
  <action name="action_30">
   <property name="text">
    <string>Консоль</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
